ansible/host_vars/slack.openobservatory.org/vault

```diff
@@ -0,0 +1,36 @@
+$ANSIBLE_VAULT;1.1;AES256
+62663863643733653839336564333933333430653137663938636535393130306463653333363065
+6166633432303964643831316665373965353733656661630a376366613634343462333930376338
+36333831333337666361366230623263653365636130656435343062323364333631313632396138
+6532313138306362380a343834663336666462643162306236383935393733623263656564616261
+64363361623739326333643935363036663261633063316334396135633433353463623862303366
+36343437666361666164646239313439373437663539383937623733353335323163666534386364
+39316361626439333062383634633165346534333665616530656663383965383738376534613738
+61346363616131346565653930333039373637646662303464326135373839613035343631656462
+66373930373336623833326361356333326161336462343033663939646430346138313132366135
+36633165653966383932613036616466396562376439373866303466623235386234386562303162
+64613665363436666130613061613136363262613039653931363965646333323436373030346661
+33633730666163326634666232646364313231383036393835353434336363636664333461313632
+37616466386434633633633839303438343236346561363866636336653364646138613638303038
+62353365306632623064373763376566303863396562643163336435303133613633643231666463
+34323130393731663962323465616134626235646636663761636661393331666365303033393866
+38383234336361303765393539623333646338626432653066646436646239303662306334656135
+35363763666233613636366533373238393837633338313933326664336166373136303735613734
+34653266336239623165393137616539326337363762366263393263666365633337373961333032
+34373666646539613939333134626463376239633562663730613963363938333034646338613463
+37326337376239333465313233343865306630326434333664633637386661346235343362316432
+35623839323338306162366366396162393133306434626466366664653733353763326363306331
+64613239383330646431663037636665383361323435636636613962366361656635316430623162
+31316363623731633930326361346464623130346232626564373939643066363938633433643333
+39356261316330626239653266613033353230623739333965333239633433613364626564343061
+30353734633239323263373137346663643762386139333039633130386665346436343239636136
+31363131666439333536356237653831666639636637363936383466663830656234623137623632
+38643764343939303465306666376566366437666635306539343832373937353166663866323161
+35623539666439616330333663656561616464666364656237343736343037616434306363646263
+62383538343365316432353965393266303266326338363263623038626634343537343262636532
+62386162356462326235373931353633386539633834353366323530313838613632323534373562
+36306465343337643235353032313061623966386234623831653633393361616535623131333265
+32353036386431386365393164336139633232376337313831666635386665333830353932343538
+38656465383838386262623765363538346533646635326637363161663362633138323436623335
+65383736343665636563633633386636306661346236613337643265643639326162613831663138
+336533656561376336376131643033303036
```

Comment:
My bad I was assuming it had been tested before being flagged as ready for review.

I checked and it seems that it contains in fact invalid YAML markup:

```
variable: |
{
  "foo": "bar"
}
```
instead of
variable: |
  {
    "foo": "bar"
  }
```

I will push a fix to the integration branch.